NEWS.md

```diff
@@ -19,6 +19,7 @@
 ## misc
 
 - Add `configureMlr()` to `.onLoad()`, possibly fixing some edge cases (#2585) (@pat-s, #2637)
+- Allow a custom threholding function to be passed to filterFeatures and makeFilterWrapper (#2686)
```

Comment:
```suggestion
- Allow a custom threholding function to be passed to `filterFeatures()` and `makeFilterWrapper()` (@annette987, #2686)
```

Please move it to the `##filters - general` section of the NEWS file

Reply:
Ok done.